src/routes/safe/components/Apps/components/AppsList.tsx

```diff
@@ -110,40 +120,50 @@ const AppsList = (): React.ReactElement => {
         </Col>
       </Menu>
       <ContentWrapper>
-        <CardsWrapper>
-          <AppCard iconUrl={AddAppIcon} onClick={openAddAppModal} buttonText="Add custom app" iconSize="lg" />
-
-          {appList
-            .filter((a) => a.fetchStatus !== SAFE_APP_FETCH_STATUS.ERROR)
-            .map((a) => (
-              <AppContainer key={a.url}>
-                <StyledLink key={a.url} to={`${appsPath}?appUrl=${encodeURI(a.url)}`}>
-                  <AppCard isLoading={isAppLoading(a)} iconUrl={a.iconUrl} name={a.name} description={a.description} />
-                </StyledLink>
-                {isCustomApp(a.url, appList) && (
-                  <IconBtn
-                    title="Remove"
-                    onClick={(e) => {
-                      e.stopPropagation()
-
-                      setAppToRemove(a)
-                    }}
-                  >
-                    <Icon size="sm" type="delete" color="error" />
-                  </IconBtn>
-                )}
-              </AppContainer>
-            ))}
-        </CardsWrapper>
-
-        <IconText
-          color="secondary"
-          iconSize="sm"
-          iconType="info"
-          text="These are third-party apps, which means they are not owned, controlled, maintained or audited by Gnosis. Interacting with the apps is at your own risk. Any communication within the Apps is for informational purposes only and must not be construed as investment advice to engage in any transaction."
-          textSize="sm"
-        />
+        <SearchInputCard value={appSearch} onValueChange={(value) => setAppSearch(value.replace(/\s{2,}/g, ' '))} />
+        {noAppsFound && <NoAppsFound query={appSearch} onSearchClear={() => setAppSearch('')} />}
+
+        <AnimatePresence>
```

Comment:
Maybe [`initial={false}`](https://www.framer.com/docs/animate-presence/##suppressing-initial-animations) would help?

Reply:
`initial` is for opacity animation when the component is mounted/unmounted; I can disable it, but reordering animation will still be there on mount

Reply:
I see...